client/src/components/LikesList.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <div>
+    <q-dialog v-model="info.show" :position="position">
+      <q-card>
+        <q-bar class="dialog-header text-white">
+          <div class="text-h6">Likes</div>
+          <q-space />
+
+          <q-btn dense flat icon="close" v-close-popup>
+            <q-tooltip>Close</q-tooltip>
+          </q-btn>
+        </q-bar>
+
+        <q-list>
+          <q-item v-for="like in getLikes" :key="like.id" class="q-my-sm" clickable>
+            <q-item-section avatar @click="$router.push({ path: `/profile/${like.profile_id}` })">
+              <q-avatar color="primary" text-color="white">
+                <img :src="like.avatar_url" />
+              </q-avatar>
+            </q-item-section>
+
+            <q-item-section @click="$router.push({ path: `/profile/${like.profile_id}` })">
+              <q-item-label>{{ like.nickname }}</q-item-label>
+            </q-item-section>
+
+            <q-item-section side>
+              <q-btn
+                class="follow-btn"
+                v-if="!like.followed"
+                @click="follow(like)"
+                unelevated
+                rounded
+                size="sm"
+                label="Follow"
+              />
+              <q-btn
+                class="unfollow-btn"
+                v-if="like.followed"
+                @click="unfollow(like)"
```

Comment:
Why you need two similar functions, please figure out how you can group them into one.